src/atomicdex/constants/qt.actions.hpp

```diff
@@ -1,14 +1,28 @@
+/******************************************************************************
+ * Copyright © 2013-2019 The Komodo Platform Developers.                      *
```

Comment:
Would be nice to have a script that update license to 2021 on every file

Reply:
implemented in https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/640/commits/5ae79ed9e972aeeee9d23ec1ba3b833e3131cfc2